samples/zigbee/cli/README.rst

```diff
@@ -0,0 +1,153 @@
+.. _zigbee_cli_sample:
+
+Zigbee: CLI
+###########
+
+The Zigbee CLI sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the Command Line Interface inside Zephyr's :ref:`zephyr:shell_api` for interaction.
+
+The sample can be used for several purposes, including:
+
+* Initial configuration of the network - forming a network as a coordinator, adding devices to the network with the install codes, setting the extended PAN ID
+* Troubleshooting the existing network - reading and writing attributes, subscribing to the reports of the parameters
+* Benchmarking - measuring time needed for message to travel from one node to another
+
+Available CLI interfaces
+************************
+
+The CLI can be accessed by using the following interfaces:
+
+* UART
+* RTT
+
+These interfaces are completely independent one from another and can be used simultaneously or exclusively.
+
+The sample demonstrates the usage of commands listed in `Zigbee CLI Reference`_.
+Zigbee CLI is integrated into the system shell accessible over serial connection.
+
+You can also use the external `Python CLI wrapper`_ to control Zigbee network and devices using Python language.
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set8_start
+   :end-before: set8_end
+
+Use two compatible development kits programmed with this sample.
+Additionally, use either one compatible development kit programmed with :ref:`zigbee_network_coordinator_sample` for testing communication or a third compatible development kits programmed with the Zigbee CLI sample.
+
+User interface
+**************
+
+All the interactions with the application are handled using serial communication.
+See `Zigbee CLI Reference`_ for the list of available serial commands.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/zigbee/cli`
```

Comment:
It is possible that path to the example will be different, we should remember to eventually change it

Reply:
OK